rerun_py/pyproject.toml

```
@@ -22,7 +22,7 @@ dependencies = [
 description = "The Rerun Logging SDK"
 keywords = ["computer-vision", "logging", "rerun"]
 name = "rerun-sdk"
-requires-python = ">=3.8, <3.13"
+requires-python = ">=3.8, <3.14"
```

Can we change this to:

```suggestion
requires-python = ">=3.8"
```

As I explain [here](https://github.com/rerun-io/rerun/issues/6224#issuecomment-2157598038), this is often preferred.

---

@hgaiser thanks for reminding me about this. Feedback accepted :)

https://github.com/rerun-io/rerun/pull/7949